### ISSUE TITLE
Full & wide alignment in Gutenberg / Breadcrumb

### ISSUE THREAD

**Issue by Dianakc** [SimonPadbury/b4st#73]
Hello I'd like to suggest you to implement these 2 based on the follwogin resources:

- Full and wide alignment in Gutenberg https://bit.ly/2R0MwVl
- Breadcrumb https://bit.ly/2FvlWhF

Thanksfor this theme is the best I'm using so far :)

**Comment by SimonPadbury** [comment]
Hi Diana,

Thankyou for your kind words :) It's time refresh **b4st**. I was just going to update it with the latest Bootstrap, but I will also make these modifications.

I have used Dimox bredcrumb on other projects, but I didn't add it to **b4st** because it is intended merely as a starter. (Not many people want breadcrumbs, these days.) But now I will install it. And I will wrap it in [theme hooks](https://github.com/SimonPadbury/b4st/wiki/b4st-Theme-Hooks) so that people can easily remove it from a child theme if they don'twant breadcrumbs. And I will instruct how to do that.

For the full and wide alignment in Gutenberg, I have used that on one project but I didn't put it in **b4st** because a lot of my historic repeat users of this theme like to have a sidebar. (I have used a variant of this theme with no sidebar and wide blocks, which I like.) I will see what I can do.

Plus, I will update Bootstrap and jQuery. I see that Bootstrap has a new icon font, so I will think about switching to using this instead of Font Awesome. Maybe I will provide both these icon fonts, so that developers can use either/or.

I see a few minor fixes are required (e.g. in `index.php` pagination), after bumping up to Bootstrap 4.4. So I will do these before I upload the next version of **b4st**.

I use SCSS a lot in other projects. I am thinking about adding Gulp + Sass to **b4st**. Or maybe I will just add a tutorial in [https://github.com/SimonPadbury/b4st/wiki](https://github.com/SimonPadbury/b4st/wiki) showing how to do this.
